### PR TITLE
ENDOC-549 Add React context params tutorial

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

File: vuepress/docs/next/tutorials/create/mfe/context-params.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- [A working instance of Entando](../../../docs/getting-started/)
+- [A configurable React MFE](./widget-configuration.md)
+
+## Configure the MFE to accept and display context parameters
+This tutorial starts where the [configurable React MFE tutorial](./widget-configuration.md) ends since many of the changes required to enable context parameters are also required when preparing a config MFE. Those changes include modifying the custom element to accept the `config` JSON from Entando, enabling a local test setup using `mfe-config.json`, and configuring the bundle descriptor `entando.json`. 
+
+1. Edit the `simple-mfe/src/App.js`. Start by mapping the `contextParams` from the `config` structure:
+```js
+const { contextParams, params} = config || {};
+```
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/create/mfe/context-params.md:27]: This is the only section that gave me trouble doing the tutorial. I added this line and did not remove the other params line. I'm guessing programmers would understand what the 'config' structure hints at but just wanted to let you know.

File: vuepress/docs/next/tutorials/create/mfe/context-params.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+::: tip
+Context params are provided on the server side via the `@wp.page` and `wp.info` custom tags. Note that the `wp.info` tag retrieves values by key for the info context as well as system parameters when the key is `systemParam`. See the [Core Tag Library](../../../docs/reference/freemarker-tags/freemarker-core-tags.md) for more information. 
+:::
+
+## Prerequisites
+- [A working instance of Entando](../../../docs/getting-started/)
+- [A configurable React MFE](./widget-configuration.md)
+
+## Configure the MFE to accept and display context parameters
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/create/mfe/context-params.md:21]: should the second level headers be capitalized since it shows up in the nav bar

**Review by Lyd1aCla1r3 — APPROVED** [review]

File: vuepress/docs/next/tutorials/create/mfe/context-params.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+2. Access context parameters from the Entando Application
+3. Set up API claims so MFEs are automatically connected to microservices in the same bundle or namespace
+  
+This tutorial will demonstrate the second technique by making use of the most commonly used context parameters:
+* `page_code`: the code of the current page, e.g. `home` or `my-page`
+* `info_currentLang`: the current language, e.g. `en` or `it`
+* `systemParam_applicationBaseURL`: the full base URL of the Entando Application
+
+::: tip
+Context params are provided on the server side via the `@wp.page` and `wp.info` custom tags. Note that the `wp.info` tag retrieves values by key for the info context as well as system parameters when the key is `systemParam`. See the [Core Tag Library](../../../docs/reference/freemarker-tags/freemarker-core-tags.md) for more information.
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/next/tutorials/create/mfe/context-params.md:14]: Yep. Might be too much detail even for a tip but best I could come up with.

File: vuepress/docs/next/tutorials/create/mfe/context-params.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/create/mfe/context-params.md:27]: I'm guessing you had to delete the other params line to get it to build correctly? It shouldn't have allowed the original line with this modified. I'll reword it.

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**